config/yurtctl-servant/setup_edgenode

```diff
@@ -94,6 +83,41 @@ error() {
     echo "$(date +"%m/%d/%Y-%T-%Z") [YURT_SERVANT] [ERROR] $@"
 }
 
+check_addr()
+{
+    echo $1|grep "^\(https\|http\)\:\/\/[0-9]\{1,3\}\.\([0-9]\{1,3\}\.\)\{2\}[0-9]\{1,3\}\:[0-9]\{1,5\}$" > /dev/null;
+    if [ $? -ne 0 ]
+    then
+        log "apiserver addr $1 is error."
+        return 1
```

Comment:
maybe we need exit 1 to stop edge revert?

Reply:
Adjusted